docs/tables/aws_drs_source_server.md

```diff
@@ -0,0 +1,63 @@
+# Table: aws_drs_source_server
+
+AWS Elastic Disaster Recovery (AWS DRS) minimizes downtime and data loss with fast, reliable recovery of on-premises and cloud-based applications using affordable storage, minimal compute, and point-in-time recovery.
+
+## Examples
```

Comment:
Would adding an example using a `where` clause with the `last_launch_result` column be helpful, e.g., `### List source servers that failed last recovery launch`? Are there any other columns we could use in additional examples to show the health/status or security information of a source server?

Reply:
@cbruno10 I have added two examples using the where clause.
- Using `last_launch_result ` field as you suggested.
- Using the `data_replication_state` field will help identify which source servers are disconnected from the data replication process.